<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookmarkbox</class>
 <widget class="QWidget" name="bookmarkbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>來源(Source)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextBrowser" name="sourcePath"/>
      </item>
      <item>
       <widget class="QPushButton" name="sourceBtn">
        <property name="text">
         <string>選擇來源(Choose)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>書籤(bookmark)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListView" name="bookmarkView"/>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="saveBtn">
           <property name="text">
            <string>儲存(Save)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addfromhisBtn">
           <property name="text">
            <string>加入緩存(add from history buffer)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="chooseBtn">
           <property name="text">
            <string>選擇(choose)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
